File: Maths_And_Stats/Algebra/Factorial/factorial.php
<html>  
<head>  
<title>Factorial Program using loop in PHP</title>  
</head>  
<body>  
<form method="post">  
    Enter the Number:<br>  
    <input type="number" name="number" id="number">  
    <input type="submit" name="submit" value="Submit" />  
</form>  
<?php   
    function fact ($n)  
    {  
        if($n <= 1)   
        {  
            return 1;  
        }  
        else   
        {  
            return $n * fact($n - 1);  
        }  
    }  
    if($_POST){  
        $fact = 1;  
        //getting value from input text box 'number'  
        $number = $_POST['number'];  
        echo "Factorial of $number:<br><br>";  
        echo fact($number) . "<br>";  
    }  
?>  
</body>  
</html>  

File: Maths_And_Stats/Algebra/Factorial/factorial.go
package main  
  
import (  
 "fmt"  
)  
  
func IterativeFactorial(number int) uint64 {  
 var result uint64 = 1  
 if number < 0 {  
  
 } else {  
  for i := 1; i <= number; i++ {  
   result *= uint64(i)  
  
  }  
 }  
 return result  
}  
func main() {  
 var number int  
 fmt.Print("Enter Number:")  
 fmt.Scanln(&number)  
 fmt.Printf("Factorial of %d = %d", number, IterativeFactorial(number))  
}  
